src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTest.java

```diff
@@ -137,6 +170,19 @@ private static Stream<Arguments> validCodecToCQLTestCases() {
             TEST_DATA.STRING_WITH_3BYTE_UTF8_CHAR));
   }
 
+  private static Stream<Arguments> validCodecToCQLTestCasesOther() {
+    // Arguments: (CQL-type, from-caller, bound-by-driver-for-cql
+    // Textual types: ASCII, TEXT (VARCHAR is an alias for TEXT).
+    return Stream.of(
+        // Binary: to be added when we have a codec for it
+        Arguments.of(
+            DataTypes.BLOB,
+            EJSONWrapper.from(
+                EJSONWrapper.EJSONType.BINARY.key(),
+                JsonNodeFactory.instance.textNode(TEST_DATA.BASE64_PADDED_ENCODED_STR)),
+            ByteBuffer.wrap(TEST_DATA.BASE64_PADDED_DECODED_BYTES)));
```

Comment:
I'll add check for zero-length case and see what missing padding gives us.
Jackson can support latter case, but not sure we want to allow that.

Reply:
Added 2 suggested tests to verify that:

1. Empty String gets decoded as empty `byte[]`/`ByteBuffer`
2. Missing padding leads to exception being thrown (if we want to change to allow padding to be optional, easy to do, FWTW)

Reply:
thanks, do we know what the actual RFC spec is we are expecting so I can include it in docs see https://en.wikipedia.org/wiki/Base64#Variants_summary_table

Reply:
Yeah, RFC 2045, aka "MIME with Padding". We can choose to enforce padding or not; right now code expects them. Since they are JSON Strings, no linefeeds added or expected (since in Strings those are to be escaped and get all ugly).

I'll see where I can add a note on this choice -- will add in spreadsheet if not yet there.